src/main/java/frc/robot/GlobalConstants.java

```diff
@@ -31,8 +31,9 @@ public static class Controllers {
 
 		public static final XboxController DRIVER_CONTROLLER = new XboxController(0);
 		public static final XboxController OPERATOR_CONTROLLER = new XboxController(1);
-		public static final GenericHID FIGHT_STICK = new GenericHID(2);
-		public static final XboxController TEST_CONTROLLER = new XboxController(3);
+		public static final GenericHID FIGHT_STICK_1 = new GenericHID(2);
```

Comment:
making sure everything's plugged into the DS in the right order is gonna go crazy lmao